docs/reference/nuspec.md

```diff
@@ -360,6 +360,9 @@ The `<group>` element without a `targetFramework` attribute is used as the defau
 > [!Important]
 > The group format cannot be intermixed with a flat list.
 
+> [!Note]
+> The format of TFM identifier used in `assembly paths` is different when compared to the TFM identifier used in `dependency groups`. If the target frameworks declared in the dependencies group of the `nuspec` and the `lib/ref` folder do not have exact matches then `pack` command will raise [NuGet Warning NU5128](../reference/errors-and-warnings/nu5128.md).
```

Comment:
Never use an acronym without it being defined earlier in the page. Change the first reference of TFM to "Target Framework Moniker (TFM), and make it a link to our target frameworks doc.

Reply:
I woudln't call it TFM identifier either. 

Because Target Framework Identifier exists and it might be confusing.

Reply:
fixed in the latest commit.